### PR TITLE
prayer: Add HD Prayer Bar

### LINKED ISSUE #8470: Add High Detail Prayer Bar
to match the high detail health bar.

alt text

### PR DESCRIPTION
**Pull request by Hydrox6**
Closes #8470
full-bar
half-bar
flicker

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerBarOverlay.java [hunk 1]
```diff
@@ -48,13 +50,17 @@
...
 	private static final Dimension PRAYER_BAR_SIZE = new Dimension(30, 5);
+	private static final int HD_PRAYER_BAR_PADDING = 1;
 
 	private final Client client;
 	private final PrayerConfig config;
 	private final PrayerPlugin plugin;
 
 	private boolean showingPrayerBar;
 
+	private final BufferedImage frontBar = ImageUtil.getResourceStreamFromClass(getClass(), "front.png");
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerBarOverlay.java]: Make these `static`

**Comment by Adam-** [comment]
Maybe this should just be a toggle instead of assuming the health bar override means you want to override the prayer bar?

**Comment by deathbeam** [comment]
It should be tied to "hd" bar, any combination of "hd"/non-"hd" hp/prayer bar looks bad

**Comment by Hydrox6** [comment]
Updated to work with the new Healthbars

**Pull request merged**